OctoAwesome/OctoAwesome.Client/Screens/CrewMemberScreen.cs

```diff
@@ -60,8 +60,8 @@ public CrewMemberScreen(ScreenComponent manager, CrewMember member) : base(manag
                 Padding = new Border(0, 0, 10, 0)
             };
             if (member.PictureFilename == null)
-                profileImage.Texture = assets.LoadTexture(typeof(CrewMember), "base");
-            else profileImage.Texture = assets.LoadTexture(typeof(CrewMember), member.PictureFilename);
+                profileImage.Texture = assets.LoadTexture(typeof(CrewMember), "Crew.base");
+            else profileImage.Texture = assets.LoadTexture(typeof(CrewMember), "Crew." + member.PictureFilename);
```

Comment:
Line Break after else and use the strning interpolation, because it can be better optimized in future .net versions
```suggestion
            else 
                profileImage.Texture = assets.LoadTexture(typeof(CrewMember), $"Crew.{member.PictureFilename}");
```